docs/tools/cli/service.rst

```diff
@@ -650,7 +650,7 @@ Updates the settings for an Aiven service.
     avn service update big-service        \
       --plan business-4     
 
-**Example:** Update the service named ``secure-database`` to only accept connections from the range ``10.0.1.0/24`` and the IP ``10.25.10.12``.
+**Example:** Update the service named ``secure-database`` to only accept connections from the range ``10.0.1.0/24`` and the IP ``10.25.10.12``. Note that there is no whitespace between the IP addresses in the command below.
```

Comment:
To make notes and warnings more visible, we use special small sections designed for it.
You can use `.. note::` for this. For example in this case if you move the note under the command line block it can be:

```
.. note::

There is no whitespace between the IP addresses in the command above.
```

It would be great if we align notes as they are used in other places. Could you do this?

Reply:
I applied this change myself to keep things moving.